net/DevExtreme.AspNet.Data.Tests/EFSortingTests.cs

```diff
@@ -45,15 +38,33 @@ class TestClass_Nullable : TestClass_Base {
             public int? Sortable { get; set; }
         }
 
+        class TestClass_CodeFirstConv1 {
```

Comment:
For anybody who is curious what `Conv` means there is an MSDN link ([here](https://github.com/DevExpress/DevExtreme.AspNet.Data/pull/83/files/7d88a127f1353b7044292e0f3b65ca9021acdb2e#diff-a62b9cb2d97007d84eff8ea2841f3be5R64))